sdcm/utils/remote_logger.py

```diff
@@ -299,6 +300,34 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"
 
 
+class KubernetesWrongSchedulingLogger(CommandClusterLoggerBase):
+    restart_delay = 30
```

Comment:
It is too frequent. 2 minutes would be enough.

Reply:
changed

Reply:
done